README.md

```diff
@@ -76,6 +76,31 @@ An instance runner job executes an action on an instance.  Currently supported a
 }
 ```
 
+### service
```

Comment:
maybe call it `container service`?

Reply:
i welcome the future "i told you so" but i think this is fine for now 😆